packages/angular/stories/tooltip.stories.ts

```diff
@@ -0,0 +1,18 @@
+import { type Meta, type StoryObj, moduleMetadata } from '@storybook/angular';
+import { TooltipDirective } from '../src/tooltip.directive';
+import { ButtonComponent } from '../src/wrappers';
+
+export default {
+  title: 'Tooltip',
+  decorators: [
+    moduleMetadata({
+      imports: [ButtonComponent, TooltipDirective]
+    })
+  ]
+} as Meta;
+
+export const Basic: StoryObj = {
```

Comment:
Maybe it will be more clear to name the story like `Tooltip directive`? We have also separated tooltip in wrappers, so maybe it could be not clear for all people?